app/models/access_token.rb

```diff
@@ -1,4 +1,8 @@
 class AccessToken < ApplicationRecord
+  TIMESTAMP_FORMAT = '%FT%T%:z'.freeze
+  PAST_TIME = DateTime.new(1970).utc.freeze
```

Comment:
Why create a DateTime object but later compare with a `Time` object? Btw another approach could be
```suggestion
  PAST_TIME = Time.at(0)
```
I regret I didn't use in my PR.

Reply:
https://github.com/3scale/porta/pull/3939/commits/963fc838f8d3e071cfd1508dd5b07d1aac2bec06